runtime/j9vm/j9vmnatives.xml

```diff
@@ -193,7 +193,10 @@ SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0-only WITH Classpath-ex
 		<export name="_JVM_HoldsLock@12"/>
 		<export name="_JVM_InitProperties@8"/>
 		<export name="_JVM_ArrayCopy@28"/>
-		<export name="_JVM_DoPrivileged@20"/>
+		<export name="_JVM_DoPrivileged@20">
+			<!--<exclude-if condition="spec.java9"/> worked with just this-->
+			<include-if condition="spec.win_.*"/>
+		</export>
```

Comment:
I believe `exclude-if` must appear after `include-if`; see https://github.com/eclipse-openj9/openj9/blob/master/sourcetools/com.ibm.uma/com/ibm/uma/om/parser/module.xsd#L72. We want both conditions else UMA builds would be broken for Java 11+.

Reply:
Thanks! I was just going to ask you about this. I'll try that.

Reply:
Thanks! I'll try that.

Reply:
This worked, the current changes compile successfully. http://vmfarm.rtp.raleigh.ibm.com/build_info.php?build_id=85791